examples/config/cockroach

```diff
@@ -1,3 +1,2 @@
 cockroach.host=cockroachdb-proxy
 cockroach.port=26257
-cockroach.address=cockroachdb-proxy:26257

```

Comment:
Why do we no longer need/want this?

Reply:
It's not referenced anywhere under `/examples`.

I can imagine clients where this can be passed in directly, instead of separate `host` and `port`, but then let's add such an example.

Reply:
> It's not referenced anywhere under /base or /examples.

:+1: thanks

pedanticisim: it would be nice if the reason for the removal were noted in the Git history